draft-ietf-rats-architecture.md

```diff
@@ -718,6 +718,15 @@ could establish trust in a Verifier as discussed above.
 
 ## Attester
 
+Attesters prove their trustworthiness to Verifiers via key material that is established in their Attesting Environments by an Attester manufacturer and/or an Endorser.
+The proof is by the signing or authenticating of Evidence. See {{freshness}}.
+The Attester manufacturer and/or Endorser only puts this key material into Attesters that it trusts.
+This part of the Attester is often known as, or performed by, a root of trust.
+The security strength or value of the overall Attestation depends greatly on the security strength of the implementation of the root of trust.
+
+There is also a non-remote attestation use case where the Attester and Verifier are directly connected such that cryptographic proof of trustworthiness is unnecessary (e.g., they are on the same chip).
```

Comment:
I am not comfortable wit using the phrase "non-remote attestation". The reader has no reference or context here and I find it quite confusing.